src/coll/mod.rs

```diff
@@ -1444,6 +1444,14 @@ pub struct Namespace {
 }
 
 impl Namespace {
+    /// Construct a `Namespace` with the given database and collection.
+    pub fn new(db: impl Into<String>, coll: impl Into<String>) -> Self {
```

Comment:
There's some minor added convenience with the `impl Into<String>` arguments, i.e. you can do `Namespace::new("foo", "bar")` rather than `Namespace { db: "foo".to_string(), coll: "bar".to_string() }`.